pkg/services/x509/uzi_validator.go

```diff
@@ -0,0 +1,94 @@
+package x509
+
+import (
+	"crypto/x509"
+	"fmt"
+	"strings"
+
+	"github.com/lestrrat-go/jwx/jwa"
+
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+	"github.com/nuts-foundation/nuts-auth/pkg/services"
+)
+
+var UziAttributes = []string{
```

Comment:
Missing docs on all exported vars.
Could you add a link to the uzi documentation?